frontend/src/api/djangoAPI.js

```diff
@@ -1,5 +1,59 @@
 import axios from 'axios';
 
-export default axios.create({
+/**
+ * initialize the client with the base url
+ */
+const http = axios.create({
     baseURL: process.env.REACT_APP_DJANGO_API
-})
\ No newline at end of file
+})
+
+class djangoAPI {
```

Comment:
This is nice, however I see it a bit incomplete,
Would be nice to get a class for a Pager, a Video, etc...